src/layouts/index.jsx

```diff
@@ -7,8 +7,13 @@ import '../styles/base.scss';
 const TemplateWrapper = ({ children }) => (
   <div>
     <Navbar />
-    <div>
-      {children}
+    <div
+      style={{
+        margin: '0 auto',
+        paddingTop: 21,
```

Comment:
would prefer this to be in a .scss file -- conventionally, you'd make a `$navbar-height` variable in _mixins.scss and then use that everywhere else you need to pad for it